courier/courier_dispatcher_test.go

```diff
@@ -54,24 +54,9 @@ func TestDispatchMessageWithInvalidSMTP(t *testing.T) {
 		messages, err := reg.CourierPersister().NextMessages(ctx, 10)
 		require.Len(t, messages, 1)
 	})
-
-	t.Run("case=max retries reached", func(t *testing.T) {
```

Comment:
Where did this test move to?

Reply:
I moved the logic of `abandoning` messages back to `DispatchQueue`, so this can't be tested here anymore. Instead the existing test at https://github.com/ory/kratos/pull/2914/files#diff-0b4d4586068f4dbc5b91bea49de83394f45726e7525e01a1c315cd2046562827R59 tests this now.